near-contract-standards/src/non_fungible_token/core/core_impl.rs

```diff
@@ -305,13 +305,41 @@ impl NonFungibleToken {
 
     /// Mint a new token without checking:
     /// * Whether the caller id is equal to the `owner_id`
+    /// * `refund_id` will transfer the left over balance after storage costs are calculated to the provided account.
```

Comment:
You might want to change this comment, since there is no `refund_id` anymore as a param